pkg/controller/volume/attacherdetacher/attacher_detacher.go

```diff
@@ -119,15 +119,23 @@ func (ad *attacherDetacher) generateAttachVolumeFunc(
 
 		if attachErr != nil {
 			// On failure, just log and exit. The controller will retry
-			glog.Errorf("Attach operation for %q failed with: %v", volumeToAttach.VolumeName, attachErr)
+			glog.Errorf(
+				"Attach operation for device %q to node %q failed with: %v",
+				volumeToAttach.VolumeName, volumeToAttach.NodeName, attachErr)
 			return attachErr
 		}
 
+		glog.Infof(
+			"Successfully attached device %q to node %q. Will update actual state of world.",
+			volumeToAttach.VolumeName, volumeToAttach.NodeName)
+
 		// Update actual state of world
 		_, addVolumeNodeErr := actualStateOfWorld.AddVolumeNode(volumeToAttach.VolumeSpec, volumeToAttach.NodeName)
 		if addVolumeNodeErr != nil {
 			// On failure, just log and exit. The controller will retry
-			glog.Errorf("Attach operation for %q succeeded but updating actualStateOfWorld failed with: %v", volumeToAttach.VolumeName, addVolumeNodeErr)
+			glog.Errorf(
```

Comment:
I assume that this is not a huge deal because the node will begin reporting that the volume is mounted -- accurate?  If so, I think that it could use a comment to that effect.